server/api/crud/projects.py

```diff
@@ -187,6 +191,10 @@ def delete_project_resources(
         ):
             server.api.crud.Logs().delete_project_logs_legacy(name)
 
+        logger.debug(
```

Comment:
Why not move all the logs inside the functions? For example, here in `delete_project_alert_events `

Reply:
because functions can get called from multiple places and lost context of the story im trying to tell using the delete project resources itself